test/e2e/tls_test.go

```diff
@@ -99,9 +99,9 @@ func (data *TestData) configureTLS(t *testing.T, cipherSuites []uint16, tlsMinVe
 	}
 }
 
-func (data *TestData) checkTLS(t *testing.T, podName string, containerName string, apiserver int, apiserverStr string) {
+func (data *TestData) checkTLS(t *testing.T, podName string, containerName string, apiserver int, apiserverStr string, controllerPodIP string) {
```

Comment:
This `controllerPodIP` should be renamed accordingly.

Reply:
done